documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -52,178 +48,167 @@ public void run(String routingKey, String message) {
-| deleteStream    | (String scopeName, String streamName)                             | Remove the Stream from Pravega and recover any resources used by that Stream                                   |
-|                 |                                                                   | Note the Stream must already exist, an exception is thrown if you delete a nonexistent stream.                                                                                                              |
-|                 |                                                                   | Returns true if the stream was deleted.                                                                                                               |
-
-After line 3 in the code is finished, we have established that the Scope exists,
-we can then go on and create the Stream in lines 5-8. 
-
-The StreamManager needs 3 things to create a Stream, the Scope's name, the
-Stream's name and a StreamConfiguration.  The most interesting task is to create
-the StreamConfiguration.
-
-Like many objects in Pravega, a Stream takes a configuration object that allows
+| (static) `create` | (URI controller)                                                  | Given a URI to one of the Pravega Controller instances in the Pravega Cluster, create a Stream Manager object. |
+| `createScope`     | (String scopeName)                                                | Creates a Scope with the given name.                                                                           |
+|                 |                                                                   | Returns **_True_** if the Scope is created, returns **_False_** if the Scope already exists.                                                                                                               |
+|                 |                                                                   | This method can be called even if the Stream is already existing.                                                                                                              |
+| `deleteScope`     | (String scopeName)                                                | Deletes a Scope with the given name.                                                                           |
+|                 |                                                                   | Returns **_True_** if the scope was deleted, returns **_False_** otherwise.                                                                                                               |
+|                 |                                                                   | **Note:** If the Scope contains Streams, the `deleteScope` operation will fail with an exception.                                                                                                               |
```

Comment:
Please, do not add ` **Note:**` here, this column is precisely to add notes and comments on the corresponding operations.

Reply:
Accepted suggestion